src/compiler/types.ts

```diff
@@ -2841,6 +2864,9 @@ namespace ts {
         // eg.  module m { export class c { } } import x = m.c;
         // When this flag is specified m.c will be used to refer to the class instead of alias symbol x
         UseOnlyExternalAliasing = 0x00000002,
+
+        // Build symbol name using property accesses and element accesses, instead of an entity name
+        AllowAnyNodeKind = 0x00000004,

```

Comment:
The name `AllowAnyNodeKind` doesn't seem to align with the intended use described in the comment. Perhaps a better description or a more descriptive name would be better here.

Reply:
Updated the description